math-scala/src/main/scala/org/apache/mahout/math/algorithms/regression/LinearRegressorModel.scala

```diff
@@ -70,17 +76,43 @@ trait LinearRegressorFitter[K] extends RegressorFitter[K] {
     val varCovarMatrix = invDegFreedomKindOf * ete(0,0) * drmXtXinv
     val se = varCovarMatrix.viewDiagonal.assign(SQRT)
     val tScore = model.beta / se
-    val tDist = new org.apache.commons.math3.distribution.TDistribution(n-k)
+    val tDist = new TDistribution(n-k)
+
+    // This is the residual sum of squares for just the intercept
+    //println(" drmTarget.ncol) = " +  drmTarget.ncol)
+    val interceptCol = model.beta.length - 1
+    val targetMean: Double = drmTarget.colMeans().get(0)
+    val rssint: Double = ((drmTarget - targetMean  ).t %*% (drmTarget - targetMean)).zSum()
+    val rssmod: Double = ete.zSum()
```

Comment:
as I recall `ete` is a 1x1 matrix and you can also access the value via `ete(0,0)` see line 76.  

Since we keep doing this, it might make sense just to collect the value a little ways upstream